EventEnrichment/src/main/java/com/amazonaws/ProcessStreamPartitionPreLoadReferenceData.java

```diff
@@ -0,0 +1,47 @@
+package com.amazonaws;
+
+import com.amazonaws.operators.PartitionPreLoadEnrichmentData;
+import com.amazonaws.pojo.Customer;
+import com.amazonaws.utils.KinesisStreamInitialiser;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Properties;
+
+public class ProcessStreamPartitionPreLoadReferenceData {
+    private static final Logger LOG = LoggerFactory.getLogger(ProcessStreamPartitionPreLoadReferenceData.class);
+    private static final String DATA_STREAM_NAME = "event-data-enrichment";
+
+
+    public static void main(String[] args) throws Exception {
+        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+
+        //read the parameters specified from the command line
+        ParameterTool parameter = ParameterTool.fromArgs(args);
+
+        KinesisStreamInitialiser ksi = new KinesisStreamInitialiser();
+
+        Properties kinesisConsumerConfig = ksi.getKinesisConsumerConfig(parameter);
```

Comment:
Any reason this can't be a static method?

Reply:
Done